build.gradle

```diff
@@ -783,12 +798,72 @@ subprojects {
         }
     }
 
+    def componentMaker = objects.newInstance(ComponentMaker)
+    def neoformComponent = componentMaker.componentFactory.adhoc('neoform')
+    components.add(neoformComponent)
+
+    configurations {
+        neoformTools {
+            canBeResolved = false
+            attributes {
+                attribute(TargetJvmVersion.TARGET_JVM_VERSION_ATTRIBUTE, JAVA_TARGET)
+            }
+            neoformComponent.addVariantsFromConfiguration(it) {}
+        }
+        neoformLibraries {
+            canBeResolved = false
+        }
+        neoformRuntimeElements {
+            canBeResolved = false
+            attributes {
+                attribute(TargetJvmVersion.TARGET_JVM_VERSION_ATTRIBUTE, JAVA_TARGET)
+                attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, Usage.JAVA_RUNTIME))
+            }
+            outgoing {
+                capability "${project.group}:${rootProject.name}-dependencies:${project.version + '-' + rootProject.TIMESTAMP}"
+            }
+            neoformComponent.addVariantsFromConfiguration(it) {}
+            extendsFrom neoformLibraries
+        }
+        neoformApiElements {
+            canBeResolved = false
+            attributes {
+                attribute(TargetJvmVersion.TARGET_JVM_VERSION_ATTRIBUTE, JAVA_TARGET)
+                attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, Usage.JAVA_API))
+            }
+            outgoing {
+                capability "${project.group}:${rootProject.name}-dependencies:${project.version + '-' + rootProject.TIMESTAMP}"
+            }
+            neoformComponent.addVariantsFromConfiguration(it) {}
+            extendsFrom neoformLibraries
+        }
+    }
+
+    artifacts {
+        neoformTools makeZip
+    }
+
+    TOOLS.each { dependencies.add('neoformTools', it.version) { transitive = false } }
+
+    ['server', 'client', 'joined'].each {
+        CONFIG?.libraries?.get(it)?.each { dependencies.add('neoformApiElements', it) { transitive = false } }
+    }
+
+    dependencies {
+        neoformLibraries("net.neoforged:minecraft-dependencies:${project.version}") {
+            capabilities {
```

Comment:
This capabilities block is no longer needed now that the minecraft deps module was moved to `minecraft-dependencies` cause it's just the default capability